source/plugins/ruby/CAdvisorMetricsAPIClient.rb

```diff
@@ -304,6 +307,15 @@ def getContainerCpuMetricItems(metricJSON, hostName, cpuMetricNameToCollect, met
                         telemetryProps["multilineLanguages"] = @clusterMultilineLanguages
                       end
                     end
+                    if (!@clusterKubernetesMetadataEnabled.nil? && !@clusterKubernetesMetadataEnabled.empty?)
+                      telemetryProps["metadataEnabled"] = @clusterKubernetesMetadataEnabled
+                    end
+                    if (!@clusterKubernetesMetadataiIncludeFields.nil? && !@clusterKubernetesMetadataiIncludeFields.empty?)
```

Comment:
typo and extra "i". should be clusterKubernetesMetadataIncludeFields

please fix across all the places